api/models.py

```diff
@@ -64,3 +64,18 @@ class Task (models.Model):
 	deadline = models.DateTimeField(blank=True, null=True)
 	date_created = models.DateTimeField(auto_now_add=True)
 	date_updated = models.DateTimeField(auto_now=True)
+
+class Event (models.Model):
+
+	EVENT_STATUS = (
+		('UPCOMING', 'Upcoming events'),
+		('CURRENT', 'Todays events'),
+		('PASSED', 'Passed events'),
+	)
+
+	title = models.CharField(max_length=255, default=None)
+	start_date = models.DateTimeField()
```

Comment:
Start date includes date and time I thought this is enough. So I must do the time separately? For end date you are right, I should include it. I just took example in random events on the internet and there were no end time specified.